src/main/java/com/incognia/api/IncogniaAPI.java

```diff
@@ -325,6 +327,15 @@ public TransactionAssessment registerPayment(RegisterPaymentRequest request)
   public void registerFeedback(
       FeedbackEvent feedbackEvent, Instant timestamp, FeedbackIdentifiers identifiers)
       throws IncogniaException {
+    registerFeedback(feedbackEvent, timestamp, identifiers, Boolean.FALSE);
+  }
+
+  public void registerFeedback(
+      FeedbackEvent feedbackEvent,
+      Instant timestamp,
+      FeedbackIdentifiers identifiers,
+      Boolean dryRun)
```

Comment:
Why not use `boolean`? As we have two versions of this method, it is expected that the user specifies a value for it. This also removes the need for the null-check below.